<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Main</class>
 <widget class="QWidget" name="Form_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>attrManager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>300</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <widget class="QWidget" name="page_RsProxyMesh">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>322</width>
        <height>248</height>
       </rect>
      </property>
      <attribute name="label">
       <string>RsProxyMesh</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolBox" name="toolBox_RPMTEB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="page_RPMGeneral">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>304</width>
            <height>178</height>
           </rect>
          </property>
          <attribute name="label">
           <string>General</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>271</width>
             <height>157</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QGroupBox" name="groupBox_Dis">
                <property name="minimumSize">
                 <size>
                  <width>131</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="title">
                 <string>Display Mode</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <widget class="QRadioButton" name="radioButton_PreviewMesh">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>40</y>
                   <width>95</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Preview Mesh</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="radioButton_BoundingBox">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>20</y>
                   <width>95</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Bounding Box</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RPMDISL">
                <property name="text">
                 <string>Layer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QGroupBox" name="groupBox_Mat">
                <property name="minimumSize">
                 <size>
                  <width>131</width>
                  <height>69</height>
                 </size>
                </property>
                <property name="title">
                 <string>Material Mode</string>
                </property>
                <widget class="QRadioButton" name="radioButton_FromScene">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>40</y>
                   <width>95</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>From Scene</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="radioButton_FromProxy">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>20</y>
                   <width>95</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>From Proxy</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RPMMATL">
                <property name="text">
                 <string>Layer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_RPMOverrides">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>304</width>
            <height>178</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Overrides</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>261</width>
             <height>86</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="checkBox_RMOBJ">
                <property name="text">
                 <string>Object ID</string>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RMTES">
                <property name="text">
                 <string>Tessellation / Displacement</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RMUSE">
                <property name="text">
                 <string>User Data</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RMVIS">
                <property name="text">
                 <string>Visbility / Matte</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="checkBox_RMOBJL">
                <property name="text">
                 <string>Layer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RMTESL">
                <property name="text">
                 <string>Layer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RMUSEL">
                <property name="text">
                 <string>Layer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_RMVISL">
                <property name="text">
                 <string>Layer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_RsShape">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>322</width>
        <height>248</height>
       </rect>
      </property>
      <attribute name="label">
       <string>RsShape</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QToolBox" name="toolBox_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_RSShapeattr">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>304</width>
            <height>184</height>
           </rect>
          </property>
          <attribute name="label">
           <string>RsShapeattr</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>30</y>
             <width>160</width>
             <height>80</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="pushButton_RSHID">
              <property name="text">
               <string>Hide All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_RSSHO">
              <property name="text">
               <string>Show All</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_RSLayerOverride">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>304</width>
            <height>178</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Layer Override</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>301</width>
             <height>141</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
